fvtest/gctest/CMakeLists.txt

```diff
@@ -28,9 +28,9 @@ add_executable(omrgctest
 target_link_libraries(omrgctest
 	omrGtestGlue
 	pugixml
+	${OMR_VM_GLUE_TARGET}
```

Comment:
This strikes me as weird, we have glue without a corresponding OMR component. Is that because it's header only? Maybe this should be omrcore_glue?

Reply:
You are correct it should be omrcore_glue.  When I did my quick scan I missed that the core "VM" structures were in omrcore.